pages/linux/runuser.md

```diff
@@ -0,0 +1,23 @@
+# runuser
+
+> Run commands as a specific user and group IDs. Does not ask for password. Only useful when run as the root user.
+
+- Run command as a different user:
+
+`runuser {{user}} -c '{{command}}'`
+
+- Run command as a different user and group:
+
+`runuser {{user}} -g {{group}} -c '{{command}}'`
+
+- Start a login shell as a specific user:
+
+`runuser {{user}} -l`
+
+- Specific a shell for running or login instead of default shell:
+
+`runuser {{user}} -l -s /bin/sh`
```

Comment:
```suggestion
`runuser {{user}} -s {{/bin/sh}}`
```
Wrap shell path in tokens and remove `-l` as per the previous comment.

Reply:
Drop `-l` is OK, no need to have this option, thanks for reviewing.